python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -810,7 +807,7 @@ def _groupby_partition_agg(df, gb_cols, aggs, columns, dropna, sort, sep):
         _agg_dict
     )
     gb.columns = [_make_name(name, sep=sep) for name in gb.columns]
-    return gb
+    return gb[sorted(gb.columns)]
```

Comment:
Can we store `[_make_name(name, sep=sep) for name in gb.columns]` in a list and then return columns in sorted order?:
```python
new_columns = [_make_name(name, sep=sep) for name in gb.columns]
gb.columns = new_columns
return gb[sorted(new_columns)]